### ISSUE TITLE
Placeholder for not resolved PP

### ISSUE THREAD

**Issue by danito** [idno/Known#1029]
Hi,
for some reason in older comments I've lost the picture of the ones commenting on my post (see here : https://nxd4n.nixekinder.be/2015/nested-commentsmentions-seem-not-to-work-neither---workaround-multiple ). Is there a way to add some dummy picture when we get a 404 ?

**Comment by benwerd** [comment]
That's weird. Known caches photos, so this shouldn't happen - and indeed, those photos are local to your system. Could your uploads folder have been overwritten at some point?

**Comment by danito** [comment]
possible, in the earlier days of my Known installation I did some wild mv
and cp and git pulls on top of a url change that I manually corrected in
the DB.
Normally it should not happen but here I am with some broken links :-)

On Thu, Aug 27, 2015 at 4:22 PM, Ben Werdmuller <notifications@github.com>
wrote:

> That's weird. Known caches photos, so this shouldn't happen - and indeed,
> those photos are local to your system. Could your uploads folder have been
> overwritten at some point?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/idno/Known/issues/1029#issuecomment-135450838>.
>



-- 
Daniel Nix

**Comment by benwerd** [comment]
Yep. It's a good idea to correct those. I'm thinking that might be something that the .htaccess should provide.

**Comment by paulcmal** [comment]
> I'm thinking that might be something that the .htaccess should provide.

In terms of performance and compatibility with different web servers, I would argue against using .htaccess for more stuff (if at all). It is typically complicated to maintain a set of potentially contradictory .htaccess files, and it [requires](http://wiki.nginx.org/LikeApache-htaccess) much filesystem stats/reads and parsing before the request gets actually processed.

> a url change that I manually corrected in the DB.

What kind of URL change are we talking about here? Did you change the base URL of your Known? The name of a specific user/file? I think we should try to fix the problem at the source and make it possible to change a URL the way you did without manually editing database entries and/or resorting to .htaccess.

Would that make sense to you people as well?

**Comment by danito** [comment]
I configured Known first with the url given by my hoster before setting up
the domain on my registrar. Then I made a .htacces rule to redirect all to
https. I had to change my userid, the owner of all entities and their slug.

I'm thinking that might be something that the .htaccess should provide.

In terms of performance and compatibility with different web servers, I
would argue against using .htaccess for more stuff (if at all). It is
typically complicated to maintain a set of potentially contradictory
.htaccess files, and it requires <http://wiki.nginx.org/LikeApache-htaccess>
much filesystem stats/reads and parsing before the request gets actually
processed.

a url change that I manually corrected in the DB.

What kind of URL change are we talking about here? Did you change the base
URL of your Known? The name of a specific user/file? I think we should try
to fix the problem at the source and make it possible to change a URL the
way you did without manually editing database entries and/or resorting to
.htaccess.

Would that make sense to you people as well?

—
Reply to this email directly or view it on GitHub
<https://github.com/idno/Known/issues/1029#issuecomment-135952562>.
